### DISCUSSION TITLE
3D-printed enclosure for FanPico

### DISCUSSION THREAD

**Discussion by NeuroticGamer99** [tjko/fanpico#36] [General]
I recently ordered several of the kits on eBay.  Before I go off and do my own model in Fusion 360, I was curious if anyone had already modeled an enclosure for the board.  I didn't find anything while searching on Google, Thingiverse, and Printables.

**Comment by wtip** [comment]
@silicon-ghost did you end up designing an enclosure?

**Comment by NeuroticGamer99** [comment]
I did eventually make a simple open (as in open-top) design since I was using straight headers, not angled headers.  The attached Fusion 360 file is basically the bottom shell of a full case.  I didn't need a top since my FanPico's are sitting on the bottom of my 8 bay hard drive enclosures.  I did put a bare circuit board in the model for ease of adding cutouts.  The mouse ears are for securing the corners to the bed while 3D printing.  Feel free to mod it to your needs.

![FanPico 1 3 Open Enclosure v12](https://github.com/user-attachments/assets/c7ee6af0-f87c-4855-af5e-d06afd667b70)
[FanPico 1.3 Open Enclosure v12.zip](https://github.com/user-attachments/files/17452357/FanPico.1.3.Open.Enclosure.v12.zip)

**Comment by wtip** [comment]
@tjko Here's my design for an enclosure for the FANPICO-0401D PCB v1.1
https://www.printables.com/model/1082300-fanpico-0401d-enclosure

**Comment by tjko** [comment]
Thanks!  I added section in Wiki with links to these:

https://github.com/tjko/fanpico/wiki/FanPico-Enclosures-(3D%E2%80%90Printed)
